src/Illuminate/Auth/SessionGuard.php

```diff
@@ -88,6 +89,13 @@ class SessionGuard implements StatefulGuard, SupportsBasicAuth
      */
     protected $events;
 
+    /**
+     * The timebox implementation.
+     *
+     * @var \Illuminate\Support\Timebox
```

Comment:
```suggestion
     * @var \Illuminate\Support\Timebox|null
```
Timebox can be null (see constructor)

Reply:
I have now made the constructor fallback to setting the "timebox" property to an instance of the Timebox class if it was not provided in the constructor parameter.
It was done like that a couple of other places in the code.

Will resolve convo now, but let me know if you disagree.